### PR TITLE
Parachute s ok

### PR DESCRIPTION
**Pull request by clock21am**
Could you please check it 
Still there is an issue of Bonus which is not there in others machines but only there in my system and please point me more suggestions

### CODE REVIEW

File: src/activities/parachute/ActivityInfo.qml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* GCompris - ActivityInfo.qml
+ *
+ * Copyright (C) 2015 Your Name <yy@zz.org>
```
Comment (petitlapin) [inline src/activities/parachute/ActivityInfo.qml]: PLease update with your name

File: src/activities/parachute/ActivityInfo.qml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  icon: "parachute/parachute.svg"
+  author: "Rajdeep Kaur &lt;rajdeep51994@gmail.com&gt;"
+  demo: true
+  title: qsTr("Parachute activity")
+  description: qsTr("Help Tux the parachutist land safely")
+  //intro: "put here in comment the text for the intro voice"
+  goal:qsTr("In this game, Tux the parachutist needs help to land safely on the fishing boat. He needs to allow for the wind direction and speed.")
+  prerequisite:qsTr("This board is game-oriented. No specific skills are needed to play.")
+  manual:qsTr("Hit any key or click on the plane to make Tux jump. Hit another key or click on Tux to open the parachute.")
+  section: qsTr("fun")
```
Comment (petitlapin) [inline src/activities/parachute/ActivityInfo.qml]: You should remove the qsTr there

File: src/activities/parachute/Parachute.qml [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+                        forhover.visible = true
+                    }
+                    onExited:{
+                        forhover.visible = false
+                    }
+                    onClicked:{
+                        if(!Activity.OneClick) {
+                            minitux.visible = true
+                            Activity.parachuefun()
+                            Activity.OneClick = 1
```
Comment (petitlapin) [inline src/activities/parachute/Parachute.qml]: variable names should be normalized, start with a lowercase.

File: src/activities/parachute/Parachute.qml [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+                    onEntered: {
+                        forhover.visible = true
+                    }
+                    onExited:{
+                        forhover.visible = false
+                    }
+                    onClicked:{
+                        if(!Activity.OneClick) {
+                            minitux.visible = true
+                            Activity.parachuefun()
```
Comment (petitlapin) [inline src/activities/parachute/Parachute.qml]: this function name is not intuitive about what it does.

File: src/activities/parachute/parachute.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+.pragma library
+.import QtQuick 2.0 as Quick
+.import GCompris 1.0 as GCompris 
+.import "qrc:/gcompris/src/core/core.js" as Core
+
+
+var currentLevel = 0
+var numberOfLevel = 4
+var items
+var checkPressed = 0
```
Comment (petitlapin) [inline src/activities/parachute/parachute.js]: should be booleans, and names should be normalized (upPressed...)

File: src/activities/parachute/Parachute.qml [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+                    }
+                }
+                if(((parachutOpen.y >= background.height/1.2) && (parachutOpen.y <=(boatmotion.height+background.height)))) {
+                    if((parachutOpen.x < boatmotion.x) || (parachutOpen.x > (boatmotion.x+boatmotion.width))) {
+                                   bonus.bad("tux")
+                    }
+                }
+            }
+            Image {
+                id: parachuteImage
```
Comment (petitlapin) [inline src/activities/parachute/Parachute.qml]: you should probably have only one image and handle it with a state.

**Comment by petitlapin** [comment]
Hi,
it's a really quick code review, without looking at the activity itself.
Johnny

**Comment by bdoin** [comment]
Glad to see your progress. Here are my remarks:

- In my screen resolution the boat is going too far on the land.
- If I don't click on the plane the plane is not shown again (can't let tux jump)
- the cloud is always moving left to right, it should be random
- when I click on the plane Tux falls from the left of the screen, not from the plane
- Image source must provide a sourceSize that must be * ApplicationInfo.ratio
- Too many variables in the Items object and many with non self explanatory names.

I got these error when I run it:
couldn't create image from  "./paratrooper-activity/resources/paratrooper/scenery3_background.png"
couldn't create image from  "./paratrooper-activity/resources/paratrooper/scenery3_background.png"

**Comment by clock21am** [comment]
=D i will update it soon

**Comment by bdoin** [comment]
Please open another PR when you are ready.

**Comment by clock21am** [comment]
okay :)

**Pull request closed without merging**